vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vitest/config';
+import codspeedPlugin from '@codspeed/vitest-plugin';
+
+export default defineConfig({
+  plugins: [codspeedPlugin()],
+  // ...
+});
```

Comment:
Basic config for now, will need adjustment if we decide to do testing with vitest